brainiak/io.py

```diff
@@ -59,6 +62,10 @@ def load_images_from_dir(in_dir: Union[str, Path], suffix: str = "nii.gz",
     files = sorted(in_dir.glob("*" + suffix))
     for f in files:
         yield nib.load(str(f))
+        logger.info(
+            'file %s is read',
+            f
+        )
```

Comment:
I would say you should use the debug level for the messages you added in "io.py".

Also, do not break lines if they do not exceed the maximum line length.

Finally, the other message placement, before `yield`, makes more sense, but I would change the wording to "Starting to read [...]" and put a full stop at the end of the message.